admin/views/helpdesk/school-impersonation.ejs

```diff
@@ -0,0 +1,68 @@
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+<main class="govuk-main-wrapper" id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <% if (messages && messages.info && messages.info.length) { %>
+                <div class="govuk-body govuk-info-message govuk-!-margin-bottom-7">
+                    <span><%= messages.info %></span>
+                </div>
+            <% } %>
+            <% if (err.hasError()) { %>
+                <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+                    <h2 class="govuk-error-summary__title" id="error-summary-title">
+                        You need to fix the errors on this page before continuing.
+                    </h2>
+                    <div class="govuk-error-summary__body">
+                        <p>See highlighted errors below.</p>
+                        <ul class="govuk-list govuk-error-summary__list">
+                            <% err.getUniqueFields(['dfeNumber']).map(f => { %>
+                                <% const errors = err.get(f) %>
+                                <% if (Array.isArray(errors)) { %>
+                                    <%  errors.map(e => { %>
+                                        <li><a href="#<%= f %>"><%= e %></a></li>
+                                <% }) } else { %>
+                                    <li><a href="#<%= f %>"><%= errors %></a></li>
+                                <% } %>
+                            <% }) %>
+                        </ul>
+                    </div>
+                </div>
+            <% } %>
+            <h1 class="govuk-heading-xl"><%= pageTitle %></h1>
+            <%- partial('partials/_gds_session_expiry') %>
+            <form action="/helpdesk/add-school-impersonation" id="schoolImpersonationForm" method="post">
```

Comment:
I think it would be better to prevent browsers caching dfeNumbers here.  In normal operation, this could be used hundreds of times a day.  I think it would be safer to make the user type out the whole number.

`autocomplete=off`

Reply:
@jon-shipley that's a fair point. I have added the attribute.